src/dotnet-compile/Program.cs

```diff
@@ -330,35 +328,28 @@ private static bool CompileProject(ProjectContext context, CompilerCommandApp ar
                 success &= GenerateCultureResourceAssemblies(context.ProjectFile, dependencies, outputPath);
             }
 
-            bool generateBindingRedirects = false;
-            if (success && !args.NoHostValue && compilationOptions.EmitEntryPoint.GetValueOrDefault())
-            {
-                generateBindingRedirects = true;
-                var rids = PlatformServices.Default.Runtime.GetAllCandidateRuntimeIdentifiers();
-                var runtimeContext = ProjectContext.Create(context.ProjectDirectory, context.TargetFramework, rids);
-                runtimeContext
-                    .MakeCompilationOutputRunnable(outputPath, args.ConfigValue);
-            }
-            else if (!string.IsNullOrEmpty(context.ProjectFile.TestRunner))
+            if (success)
```

Comment:
Can you extract all the code beyond this point into a separate method called something like `PublishForLocalMachine`? Ideally, for the long term changes I'd want Compile to just call the compilers and nothing else :)